README.md

```diff
@@ -27,6 +27,32 @@ You can play with the example by
 - Changing the number of cards in the array in getCardDetails() in app/src/main/kotlin/com/intuit/truffle/shuffle/MainActivity.kt
 - Changing the percentages of the view group in app/src/main/res/values/card_view_group_percentages.xml
 
+To contribute to this project, make sure that the code committed matches this project's formatting standards.
+
+To reformat or check that committed code is compliant, follow one of the following options:
+
+#### Use ktlint engine (recommended - [ktlint docs](https://ktlint.github.io/)):
+
+ - `ktlint`
+    <p> Use this command to check the formatting rules are covered </p>
+ - `ktlint "src/**/*.kt" "!src/**/*Test.kt"`
+    <p> Check only certain locations (prepend ! to negate the pattern) </p>     
+ - `ktlint -F`
+    <p> Use this command to re-format code that is not compliant with this project's formatting rules.
+    (if some errors cannot be fixed automatically they will be printed to stderr) </p> 
+ - `ktlint -F "src/**/*.kt"`
+    <p> Use this command to re-format certain locations (prepend ! to negate the pattern)  . 
+    (if some errors cannot be fixed automatically they will be printed to stderr) </p>    
+ - `ktlint --install-git-pre-commit-hook`
+    <p> Install git hook to automatically check files for style violations on commit. </p>    
+
+#### Use Gradle tasks ([ktlint docs](https://ktlint.github.io/)):
```

Comment:
I would put these before the ktlint engine section and mark this as the recommended mechanism as it is much easier for people to understand.